### PR TITLE
City and state filtering functionality

### PR DESCRIPTION
**Pull request by rahularoradfs**
## Briefly describe the feature or bug that this PR addresses.

Adds functionality to enable filtering by city and state. 

## Please link the Airtable ticket associated with this PR.

Per Julia Ma's request for CITF modelling. 

## Describe the steps you took to test the feature/bugfix introduced by this PR.

Tested locally by retrieving data from Alberta and Ontario. 
Also fixed Canadian data in Airtable to enable this sort of filtering. 

## Does any infrastructure work need to be done before this PR can be pushed to production?

The base for this ticket is Austin's rounding ticket; it can be merged once Austin's rounding ticket is merged.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -28,3 +28,9 @@ app/database_etl/tableau_data_connector/token.pickle
 
 # cached Stan models
 **/stanmodelcache*
+.vscode
+
+# CSVs folder used to sync dbs
+csvs/
+load-dotenv.sh
+push_csvs.py
```
Comment (atatmaja) [inline .gitignore:36]: Unrelated - I wonder if we should commit this script, we're likely to have to use it in the future

File: README.md [hunk 2]
```diff
@@ -153,3 +153,11 @@ Confirm that the data has indeed been migrated by checking pgAdmin 4.
 3. Click `DOWNLOAD JSON` and save the file as `credentials.json` in the `tableau_data_connector` directory.
 4. Run `table_generator.py` for the first time. Authenticate using `can.serosurveillance.dev@gmail.com` the first time you run this.
 
+# Helpful Code Snippets
```
Comment (atatmaja) [inline README.md:156]: Thanks for documenting this!

**Review by atatmaja — APPROVED** [review]
LGTM

File: .gitignore (hunk 1 above)
Comment (rahularoradfs) [reply .gitignore:36]: I agree that we should! Just wasn't sure that it should be in the parent folder or where the right spot to put it was.

File: README.md (hunk 2 above)
Comment (rahularoradfs) [reply README.md:156]: np!

**Pull request merged**